Contrib/FreeWilson/freewilson.py

```diff
@@ -489,19 +489,19 @@ def _rgroup_sort(r):
     if r[0] == "R": return ("R", int(r[1:]))
     return (r, None)
 
-def predictions_to_csv(outstream, predictions):
+def predictions_to_csv(outstream, decomposition: FreeWilsonDecomposition, predictions):
```

Comment:
I find the type hints for arguments horrible and not particularly useful. But this is contrib code, so I won't do the review on "RDKit core" standards... if you really think this is worthwhile, then I'll swallow my objections

Reply:
Type hints appear to be the new normal.  I’m ambivalent, but they do have their uses.